scripts/monitoring/delete-stuck-projects.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# This is a workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1367432
+
+# Look for any projects found during the last run of this script
+if [ -f /tmp/stuck_projects ]; then
+  projects_to_delete="$(cat /tmp/stuck_projects)"
+
+  # If the project still exists, is empty, and contains nothing, delete it.
+  for project in ${projects_to_delete}; do
+    oc get project ${project} &> /dev/null
+
+    if [ $? == 0 ] && [ -z "$(oc get all -n ${project})" ]; then
+      oc get projects ${project} -o yaml
```

Comment:
It will be stored in the cron log, but I should probably make a separate log file for it instead. I can make a change to the cron job for that part.